```
CMakeFiles/MitkLogging.dir/src/AutoplanLogging.cpp.o: In function `Logger::Log::reinitLogger() const':
/var/lib/jenkins/home/jobs/MITK-PR/workspace/Modules/Logging/src/AutoplanLogging.cpp:259: undefined reference to `boost::locale::generator::generator()'
/var/lib/jenkins/home/jobs/MITK-PR/workspace/Modules/Logging/src/AutoplanLogging.cpp:259: undefined reference to `boost::locale::generator::~generator()'
/var/lib/jenkins/home/jobs/MITK-PR/workspace/Modules/Logging/src/AutoplanLogging.cpp:252: undefined reference to `boost::locale::generator::~generator()'
CMakeFiles/MitkLogging.dir/src/AutoplanLogging.cpp.o: In function `boost::locale::generator::operator()(std::string const&) const':
/var/lib/jenkins/home/jobs/MITK-PR/workspace/build/ep/include/boost/locale/generator.hpp:202: undefined reference to `boost::locale::generator::generate(std::string const&) const'
collect2: error: ld returned 1 exit status
```

--- a/Modules/Logging/src/AutoplanLogging.cpp
+++ b/Modules/Logging/src/AutoplanLogging.cpp
@@ -256,9 +256,6 @@ namespace Logger
         % boost::log::expressions::attr<std::string>("Source")
       );
 
-      std::locale loc = boost::locale::generator()("en_US.UTF-8");
-      sink2->imbue(loc);
-
       boost::log::core::get()->add_sink(sink2);
     }
 

--- a/Modules/QtWidgets/src/QmitkStdMultiWidget.cpp
+++ b/Modules/QtWidgets/src/QmitkStdMultiWidget.cpp
@@ -1616,7 +1616,7 @@ void QmitkStdMultiWidget::HandleCrosshairPositionEventDelayed()
   itk::Index<3> p;
   mitk::BaseRenderer* baseRenderer = this->mitkWidget1->GetSliceNavigationController()->GetRenderer();
   unsigned int timestep = baseRenderer->GetTimeStep();
-  auto timeSteps = image->GetTimeSteps();
+  auto timeSteps = image.IsNotNull() ? image->GetTimeSteps() : 0;
 
   if(image.IsNotNull() && (timeSteps > timestep ))
   {

